### PR TITLE
[FEATURE] Added own implementation of `EntityListenerResolver`

### PR DESCRIPTION
**Pull request by maxbrokman**
### Changes proposed in this pull request:

Added own implementation of `EntityListenerResolver` listeners using the Laravel container

This allows listeners to have dependencies.

### CODE REVIEW

**Comment by maxbrokman** [comment]
@driesvints

**Comment by patrickbrouwers** [comment]
Docblocks, mate ;)

File: tests/Resolvers/EntityListenerResolverTest.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+use LaravelDoctrine\ORM\Resolvers\EntityListenerResolver;
+use Mockery as m;
+
+class EntityListenerResolverTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var m\MockInterface|Container
+     */
+    private $container;
+    /**
```
Comment (driesvints) [inline tests/Resolvers/EntityListenerResolverTest.php:15]: Space between properties here.

**Comment by driesvints** [comment]
Whoho! 🎉

Nice job @maxbrokman!

**Pull request merged**